compiler-rt/lib/builtins/CMakeLists.txt

```
@@ -884,7 +884,11 @@ else ()
   if(COMPILER_RT_DISABLE_AARCH64_FMV)
     list(APPEND BUILTIN_DEFS DISABLE_AARCH64_FMV)
   elseif(COMPILER_RT_BAREMETAL_BUILD)
-    list(APPEND BUILTIN_DEFS ENABLE_BAREMETAL_AARCH64_FMV)
+    foreach (arch ${BUILTIN_SUPPORTED_ARCH})
+      if("${arch}" STREQUAL "aarch64")
```

This will miss some target names which may also require `ENABLE_BAREMETAL_AARCH64_FMV` (e.g. arm64, arm64ec). I think we can use something like this instead:

```suggestion
      if("${arch}" MATCHES "arm64|aarch64")
```